Formula/r/rpm.rb

```diff
@@ -24,67 +15,56 @@ class Rpm < Formula
   end
 
   bottle do
-    rebuild 1
-    sha256 arm64_ventura: "2d97d36743ae989839e7ef4c4d3f396c2f7cd5ab7ff87b429510a4bb9b047cf7"
-    sha256 ventura:       "d5085f39aee9be16443d96ba081f46cfb80818dc6f0c3bb5367f6d79b62c51fd"
-    sha256 x86_64_linux:  "f7021265bd0654607b2e50709a8e29f0be08e3da8ca407566b04109fa0d176e8"
+    sha256 x86_64_linux: "ec816d797170c592fe18feb85c38f2ed09be140c4da9c8e48ff8c2d17cf380e6"
   end
 
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "python@3.11" => [:build, :test]
+  depends_on "acl"
+  depends_on "bzip2"
+  depends_on "dbus"
+  depends_on "elfutils"
   depends_on "gettext"
   depends_on "libarchive"
+  depends_on "libcap"
   depends_on "libmagic"
+  depends_on :linux
   depends_on "lua"
-  depends_on macos: :ventura
   depends_on "openssl@3"
   depends_on "pkg-config"
   depends_on "popt"
   depends_on "sqlite"
   depends_on "xz"
-  depends_on "zstd"
-
-  uses_from_macos "bzip2"
-  uses_from_macos "zlib"
-
-  on_macos do
-    depends_on "libomp"
-  end
+  depends_on "zlib"
 
   conflicts_with "rpm2cpio", because: "both install `rpm2cpio` binaries"
```

Comment:
It seems that despite the `depends_on :linux` condition, the conflict still needs to  be `on_linux` only. `rpm2cpio`'s `conflicts_with` is scoped.

```
  rpm
    * Formula rpm2cpio should also have a conflict declared with rpm
```